tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,208 @@
+# testImportSNV.TCGA.R
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+context("Tools for importing MAF files into a rSNV file")
+
+test_that("parameter errors are correctly handled", {
+    # fMAF is not provided
+    expect_error(importSNV())
+    # fMAF is NULL/empty vector
+    expect_error(importSNV(fMAF = NULL))
+    # fMAF input vector contains non-string values
+    expect_error(importSNV(fMAF = 123))
+    # file in fMAF could not be opened/does not exist
+    expect_error(importSNV(fMAF = c("nosuchfileMAF")))
+
+    # if rSNV parameter is provided, rSNV file does not exist
+    expect_error(importSNV(fMAF = testMAF, rSNV = "nosuchfileSNV"))
+    # if rSNV parameter is provided, but is NULL
+    expect_error(importSNV(fMAF = testMAF, rSNV = NULL))
+    # if rSNV parameter is provided, but is non-string value
+    expect_error(importSNV(fMAF = testMAF, rSNV = 123))
+    # if rSNV is given vector input
+    expect_error(importSNV(fMAF = testMAF, rSNV = c("abc", "xyz")))
+    # if rSNV is not provided, new file could not be created (file name conflict)
+    expect_error(importSNV(fMAF = testMAF))
+
+    # if na.rm is not given logical input
+    expect_error(importSNV(na.rm = 1))
+    expert_error(importSNV(na.rm = NULL))
+    expert_error(importSNV(na.rm = logical()))
+    expert_error(importSNV(na.rm = c(TRUE, TRUE)))
+
+    # if silent is not given logical input
+    expect_error(importSNV(silent = 1))
+    expert_error(importSNV(silent = NULL))
+    expert_error(importSNV(silent = logical()))
+    expert_error(importSNV(silent = c(TRUE, TRUE)))
+
+    # write.log is not given logical input
+    expect_error(importSNV(writeLog = 1))
+    expert_error(importSNV(writeLog = NULL))
+    expert_error(importSNV(wrtieLog = logical()))
+    expert_error(importSNV(writeLog = c(TRUE, TRUE)))
+})
+
+test_that("a sane input gives an expected output with differnt input parameters", {
+
+    # make small input (testIn)
+    testIn <- "~/Desktop/rete/testIn"
+    # make small output for that input manually (testOut)
+    write("# metadata \n# 1 \n# 2  \n# 3 \n# 4 \n# 5 \n# metadata END\n", "testOut")
```

Comment:
Why don't you just create a data frame trueOut? Also you'll need to turn stringsAsFactors off.